test/transformer.base.js

```diff
@@ -1446,6 +1446,37 @@ module.exports = function base(transformer, pathname, transformer_name) {
         });
       });
 
+      if ('browserchannel' !== transformer && 'uws' !== transformer)
```

Comment:
`uws` is skipped because it only truncates the message without closing the connection, see https://github.com/uWebSockets/uWebSockets/issues/196.